components/auth/Join.tsx

```diff
@@ -92,6 +94,28 @@ const Join = () => {
           error={formik.touched.password ? formik.errors.password : undefined}
           onChange={formik.handleChange}
         />
+
+
+        {/* ADDING CHECKBOX HERE */}
+        <div className="form-control flex  flex-row items-center"> {/* DIV FOR CHECKBOX  */}
+          <div className="space-x-2"> {/* Add spacing between checkbox and label */}
+            <Checkbox type="checkbox" className='checkbox checkbox-primary checkbox-xs' onChange={(e) => {
+              // Use setFieldValue to update formik.values
+              formik.setFieldValue('agreeToTerms', e.target.checked);
+            }} />
+            <span className="checkbox-toggle"></span>
+          </div>
+          <label className="label">
+            <span className="label-text">Agree to <a href='/terms-condition' className='text-cyan-600' target="_blank">Terms and conditions</a></span>
```

Comment:
- Can we use `text-primary` instead of `text-cyan-600` to make it more consistent with branding color?
- Can we use `Link` component from `next/link` instead of `a` tag?
- Kindly consider removing comments from the code for improved readability.
- There is one more place you want to add the checkbox; see `JoinWithInvitation` component.
- There seems to be a small mismatch between your prettifier and one we're using.

All other things looks good to me.

Reply:
Done